README.md

```diff
@@ -7,6 +7,17 @@ At some point in the future, these will likely be run regularly and
 on-demand as part of [a visual script-runner
 tool](https://github.com/edgi-govdata-archiving/overview/issues/149).
 
+## `archive.sh`
```

Comment:
Can we make this heading consistent with the one below in the readme? don't care which way